README.md

```diff
@@ -83,7 +83,23 @@ Docker and docker compose are the only required dependencies. You can either ins
 <p align="right">(<a href="#top">back to top</a>)</p>
 
 #### Environment Variables
-
+Environment variables can be set using a `.env` file(use `.env.template` file for reference). The following variables are used:
+
+- `DEV`: When set to `true`, stack-trace is shown with error messages.(Default: `false`)
+- `MYSQL_PASSWORD`: A secret password for the MySQL database.
```

Comment:
```suggestion
- `MYSQL_PASSWORD`: A randomly generated secret password for the MySQL database.
```

Reply:
is this randomly generated? In local setup we can set it what we want.

Reply:
I meant to say that the user should randomly generate a strong password. This is fine too.